### ISSUE TITLE
Add id to rows?

### ISSUE THREAD

**Issue by stevepowell99** [haozhu233/kableExtra#720]
Is there any way to put an id on the <row> elements of the table, eg just the row number or a custom vector? Or more generally is there any way to add custom HTML to the elements of the table like we can add CSS?

**Comment by dmurdoch** [comment]
Yes, just add it to the dataframe, and when you pass it through kable() or kbl(), make sure to use `escape = FALSE`.
